.gitignore

```diff
@@ -11,3 +11,4 @@ open-source/libusrsctp
 open-source/libwebsockets
 open-source/local
 outputs
+open-source/amazon-kinesis-video-streams-producer-sdk-cpp
```

Comment:
Are we sure we need to add this to ignore? I mean, there is a way to modify/send CR for the linked project but perhaps we don't want this now

Reply:
I don't believe we want this, @chehefen was something happening that you needed this? Maybe there is something we need to fix to make this not needed.